### DISCUSSION TITLE
One and done

### DISCUSSION THREAD

**Discussion by VeriumETH** [nftchef/art-engine#276] [Help + Q&A]
Hey @nftchef 

I hope you are having a great start to your weekend. I was just wondering if it's possible to only use a trait once? Lets say I have 1000 images I want only used once for each one. Would that be possible? Thank you in advance to anyone who helps if I do not respond promptly.

**Comment by barslz** [comment]
create 1000 different configuration for each image/trait that you want to use.

**Comment by VeriumETH** [comment]
> create 1000 different configuration for each image/trait that you want to use.

I was honestly considering that as a last resort since all of the traits are going to be named numerically in the order I want them to generate. No harm in trying it out I suppose.
